boards/nxp/mimxrt700_evk/doc/index.rst

```diff
@@ -0,0 +1,199 @@
+.. zephyr:board:: mimxrt700_evk
+
+NXP MIMXRT700-EVK
+#################
+
+Overview
+********
+
+The new i.MX RT700 CPU architecture is composed of a high-performance main compute subsystem,
+a secondary “always-on” sense-compute subsystem and specialized coprocessors.
+
+The main compute subsystem uses a 325 MHz capable Arm® Cortex®-M33 (CM33).
+Similar to the i.MX RT600 crossover MCU, the i.MX RT700 includes a Cadence Tensilica® HiFi 4 DSP.
+The HiFi 4 is a high performance DSP core based upon a Very Long Instruction Word (VLIW) architecture,
+which is capable of processing up to eight 32x16 MACs per instruction cycle. It can be used for offloading
+high-performance numerical tasks such as audio and image processing and supports both fixed-point and
+floating-point operations.
+
+Hardware
+********
+
+- Main Compute Subsystem:
+      Arm Cortex-M33 up to 325 MHz
+      HiFi 4 DSP up to 325 MHz
+      eIQ Neutron NPU up to 325 MHz
+- Sense Compute Subsystem:
+      Arm Cortex-M33 up to 250 MHz
+      HiFi 1 DSP up to 250 MHz
```

Comment:
Incorrectly formatted sub bullet list

```
- list item

  - sublist item
  - ...
```

Note the blank line needed before starting sublist

Reply:
Updated. I will check later when online doc generated.